core/src/test/scala/unit/kafka/server/ApiVersionsRequestTest.scala

```diff
@@ -84,17 +125,38 @@ class ApiVersionsRequestTest(cluster: ClusterInstance) extends AbstractApiVersio
     assertEquals(ApiKeys.API_VERSIONS.latestVersion(), apiVersion.maxVersion())
   }
 
-  @ClusterTest(metadataVersion = MetadataVersion.IBP_3_7_IV4, serverProperties = Array(
-    new ClusterConfigProperty(key = "unstable.api.versions.enable", value = "false"),
-    new ClusterConfigProperty(key = "unstable.metadata.versions.enable", value = "false"),
+  @ClusterTests(Array(
```

Comment:
please add TODO and KAFKA-xxxx

Reply:
Already added the TODO in the top of this file. L30